app.js

```diff
@@ -98,7 +98,7 @@ app.get('/api/v1/projects/:id', async (request, response) => {
 
 app.post('/api/v1/palettes', async (request, response) => {
   const pallete = request.body;
-  for (let palleteInfo of ['name', 'color1', 'color2', 'color3', 'color4', 'color5']) {
+  for (let palleteInfo of ['project_id', 'name', 'color1', 'color2', 'color3', 'color4', 'color5']) {
     !pallete[palleteInfo] ?
     response.status(422).send({ error: `Expected format: { name: <String>, color1: <String>, color2: <String>, color3: <String>, color4: <String>, color5: <String>}. You're missing a "${palleteInfo}" property.` }) : '';
```

Comment:
Hey!  looks like you need to change the error string too.  It should be `Expected format: { project_id: <Integer>, name: <String>, color1: <String>, color2: <String>, color3: <String>, color4: <String>, color5: <String>}. You're missing a "${palleteInfo}" property.`

Reply:
Thanks, looks great!